/**:
  ros__parameters:
    image_width: 736
    image_height: 736
    camera_matrix:
      rows: 3
      cols: 3
      data: [230.835588, -0.104723, 366.729170, 0.000000, 232.336174, 365.053040, 0.000000, 0.000000, 1.000000]
    distortion_model: "plumb_bob"
    distortion_coefficients:
      rows: 1
      cols: 4
      data: [-0.020724, -0.004885, 0.002493, -0.000835]
    rectification_matrix:
      rows: 3
      cols: 3
      data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
    projection_matrix:
      rows: 3
      cols: 4
      data: [230.835588, -0.104723, 366.729170, 0.000000, 0.000000, 232.336174, 365.053040, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]